tokio-macros/Cargo.toml

```diff
@@ -24,7 +24,7 @@ proc-macro = true
 [dependencies]
 proc-macro2 = "1.0.60"
 quote = "1"
-syn = { version = "2.0", features = ["full"] }
+syn = { version = "2.0", features = ["full", "extra-traits"] }
```

Comment:
What do you need this feature for? I prefer to avoid adding these features as they hurt compilation times.

Reply:
`impl PartialEq for Attribute`. I guess I could duplicate code. Let me figure out.

Reply:
I see. if it is possible, then try to perform the equality check manually. Thanks!